fuel-tx/src/transaction/policies.rs

```diff
@@ -0,0 +1,324 @@
+use fuel_types::{
+    canonical::{
+        Deserialize,
+        Error,
+        Input,
+        Output,
+        Serialize,
+    },
+    BlockHeight,
+    Word,
+};
+
+#[cfg(feature = "random")]
+use rand::{
+    distributions::{
+        Distribution,
+        Standard,
+    },
+    Rng,
+};
+
+bitflags::bitflags! {
+    /// See https://github.com/FuelLabs/fuel-specs/blob/master/src/tx-format/policy.md#policy
+    #[derive(Clone, Copy, Default, Debug, PartialEq, Eq, Hash)]
+    #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+    pub struct PoliciesBits: u32 {
+        /// If set, the gas price is present in the policies.
+        const GasPrice = 1;
+        /// If set, the witness limit is present in the policies.
+        const WitnessLimit = 2;
+        /// If set, the maturity is present in the policies.
+        const Maturity = 4;
+        /// If set, the max fee is present in the policies.
+        const MaxFee = 8;
```

Comment:
Might look better to use bitshifts or binary representation

```suggestion
        const WitnessLimit = 1<<2;
        /// If set, the maturity is present in the policies.
        const Maturity = 1<<3;
        /// If set, the max fee is present in the policies.
        const MaxFee = 1<<4;
```